Streetcode/Streetcode.WebApi/appsettings.json

```diff
@@ -51,6 +51,7 @@
     "PropertiesToIgnore": ["token", "password"]
   },
   "AllowedHosts": "*",
+  "ApplicationUrl": "https://localhost:5001",
```

Comment:
Parhaps this should be an array and we should include http url here as well, as we will run it on http inside the container